css/_atlaskit_overrides.scss

```diff
@@ -2,13 +2,13 @@
  * Move the @atlaskit/flag container up a little bit so it does not cover the
  * toolbar with the first notification.
  */
-.jIMojv{
+.atlaskit-portal > [class$="-FlagGroup"] {
     bottom: calc(#{$newToolbarSizeWithPadding}) !important;
 }
 
 /**
  * Disable the slide-in animation for @atlaskit/flag due to the animation
- * repeating for each queued flag once it becomes the top flag.
+ * repeating for each queued flag once it becomes the top flag. (Obsolete? class .mIBKA does not exist)
```

Comment:
@saghul Is it possible that some of these styles were already obsolete? I couldn't find at least this one and another one below on meet.jit.si. These are generic `styled-components` classes so they might have changed in an earlier atlaskit update that is already part of the current jitsi-meet build.

Reply:
They may have changed names, I don't remember how they are picked. You may need to inspect the current code and find the new name, in case it still applies indeed.